src/catalog/rough.ipynb

```diff
@@ -0,0 +1,181 @@
+{
+ "cells": [
```

Comment:
Should we remove this notebook? Should we add an ignore line in `.gitignore` for notebooks?